usagov/usagov-2021
# USAGOV-1365-Mobile-Sticky-Bar-hides-field: Fix the Sticky Bar from hiding content

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1365

## Description

Fix mobile sticky bar so it doesn't hide content. Added top scroll padding when the mobile nav is visible.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Using the developer tools to view the website as a mobile device, go to http://localhost/elected-officials
2. Submit the form without filling it out
3. Go to the Alert Box and click one of the links to jump to a specific part of a page.
<img width="467" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/f77ac2c6-d43a-40d7-8f62-bdfd78e10b90">

4. Check that it doesn't hide the field
<img width="397" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/4f17ae3d-d497-4729-8db2-d793807e5ee8">

5. Now check that everything is working normally on devices like computers.


### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
